parlai/core/torch_ranker_agent.py

```diff
@@ -58,6 +58,13 @@ def add_cmdline_args(cls, argparser):
             help='The source of candidates during evaluation (defaults to the same'
             'value as --candidates if no flag is given)',
         )
+        agent.add_argument(
+            '--add-label-to-fixed-cands',
```

Comment:
do we see this as generally useful?

Reply:
I am not sure; this was specifically useful for the Wikipeida IR task from the Starspace paper (which we used for the poly-encoder/bi-encoder experiments). I've made the option `hidden` to keep from the user, and only allow in certain circumstances.